frontend/public/module/k8s/cluster-settings.ts

```diff
@@ -188,6 +188,22 @@ export const getErrataLink = (cv: ClusterVersionKind): string => {
   return `https://access.redhat.com/downloads/content/290/ver=${major}.${minor}/rhel---7/${major}.${minor}.${patch}/x86_64/product-errata`;
 };
 
+// example link: https://docs.openshift.com/container-platform/4.2/release_notes/ocp-4-2-release-notes.html#ocp-4-2-4
+export const getReleaseNotesLink = (cv: ClusterVersionKind, lastCompleted: boolean): string => {
+  const version: string = lastCompleted ? getLastCompletedUpdate(cv) : getCurrentVersion(cv);
```

Comment:
Also need to get `channel` as only `fast` and `stable` channels should get a link as only they have release notes.